lib/stat_tracker.rb

```diff
@@ -39,21 +16,12 @@ def average_goals_per_game
   end
 
   def average_goals_by_season
-    sums = {}
-    averages = {}
-    @game_collection.collection.each do |game|
-      if !sums.key?(game[1].season)
-        sums[game[1].season] = (game[1].home_goals.to_i + game[1].away_goals.to_i)
-      else
-        sums[game[1].season] += (game[1].home_goals.to_i + game[1].away_goals.to_i)
-      end
+    avg_gpg = Hash.new(0)
+    @season_collection.collection.each_pair do |key, season_array|
+      season_array.find_all { |game| avg_gpg[key] += (game.home_goals.to_f + game.away_goals.to_f) }
```

Comment:
We can revisit a lot of this kind of thing later on when we do the full refactor. 20 characters over isn't going to kill us anyway. Many of the other style guides actually talk about a 130 character limit rather than an 80 character one anyway.